ui/src/main/resources/Meeting/Code/Translations.fr.xml

```diff
@@ -77,9 +77,9 @@ contrib.meeting.field.participants=Participants
 contrib.meeting.field.groups=Groupes participants
 contrib.meeting.field.notes=Compte rendu
 contrib.meeting.field.status=Statut
-contrib.meeting.field.lastEmailSent=Dernier mail envoyé sur le
+contrib.meeting.field.lastEmailSent=Dernier courriel envoyé le
```

Comment:
BTW have we had a discussion about whether we should use "courriel" vs "email" vs "mail" for the FR language? I can read on http://romy.tetue.net/faut-il-dire-mail-ou-courriel that the answer is not obvious. Personally, being French, when I read "courriel" I really find that strange and awkward and I'm immediately thinking about a "quebecisme" (even though I'm wrong since courriel seems to be the official term).

So the discussion is between being right (courriel) or being understood (email, mail, etc).

I'm fine with whatever we choose. I was just wondering if there has been a discussion because:
1) otherwise, we might have translators moving back from "courriel" to "mail" or "email"
2) We need to document our choice somewhere so that translations know what's been decided

WDYT?

Reply:
There is thread here about it: https://forum.xwiki.org/t/email-or-courriel-in-french-language/3674
It wasn't really discussed but the main argument is that "courriel" is the official word to designate e-mail in french.
Personally, I think switching to "courriel" is a good idea, it makes translations more professional in a way.

Reply:
Thanks Adel, I've commented on https://forum.xwiki.org/t/email-or-courriel-in-french-language/3674